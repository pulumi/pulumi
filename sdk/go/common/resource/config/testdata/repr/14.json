{"my:mapKey":{"a":{"bar":"blah","secure":"foo"},"c":"d"}}