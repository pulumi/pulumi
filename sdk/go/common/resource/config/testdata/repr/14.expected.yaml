decrypt:
  my:mapKey: '{"a":{"bar":"blah","secure":"foo"},"c":"d"}'
paths:
  my:mapKey:
    value:
      a:
        bar: blah
        secure: foo
      c: d
    string: '{"a":{"bar":"blah","secure":"foo"},"c":"d"}'
    redacted: '{"a":{"bar":"blah","secure":"foo"},"c":"d"}'
    object: KBAAF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAABa/4JXAAIBYRdtYXBbc3RyaW5nXWludGVyZmFjZSB7ff+CKgACA2JhcgZzdHJpbmcMBgAEYmxhaAZzZWN1cmUGc3RyaW5nDAUAA2ZvbwFjBnN0cmluZwwDAAFk
    secure: false
    isObject: true
  my:mapKey.a:
    value:
      bar: blah
      secure: foo
    string: '{"bar":"blah","secure":"foo"}'
    redacted: '{"bar":"blah","secure":"foo"}'
    object: KBAAF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAAAt/4IqAAIDYmFyBnN0cmluZwwGAARibGFoBnNlY3VyZQZzdHJpbmcMBQADZm9v
    secure: false
    isObject: true
  my:mapKey.a.bar:
    value: blah
    string: blah
    redacted: blah
    object: ERAABnN0cmluZwwGAARibGFo
    secure: false
    isObject: false
  my:mapKey.a.secure:
    value: foo
    string: foo
    redacted: foo
    object: EBAABnN0cmluZwwFAANmb28=
    secure: false
    isObject: false
  my:mapKey.c:
    value: d
    string: d
    redacted: d
    object: DhAABnN0cmluZwwDAAFk
    secure: false
    isObject: false
